From 2f892f4c6bf56e585a624b12b77b992eabbd52e2 Mon Sep 17 00:00:00 2001
From: Christian Grothoff <christian@grothoff.org>
Date: Sun, 4 Feb 2024 18:28:44 +0100
Subject: small typo likely from copy&paste causing macro names to be different
 between builds with and without messages resulting in compilation failure if
 configured with --disable-messages

Signed-off-by: Jan Palus <jpalus@fastmail.com>
---
 src/microhttpd/connection.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/microhttpd/connection.c b/src/microhttpd/connection.c
index 2bb37f7f..5f310c8e 100644
--- a/src/microhttpd/connection.c
+++ b/src/microhttpd/connection.c
@@ -381,7 +381,7 @@
   "<body>HTTP request has invalid character in footer.</body>" \
   "</html>"
 #else
-#define ERR_RSP_INVALID_CHR_IN_HEADER ""
+#define ERR_RSP_INVALID_CHR_IN_FOOTER ""
 #endif
 
 /**
@@ -394,7 +394,7 @@
   "<body>HTTP request header line has no colon character.</body>" \
   "</html>"
 #else
-#define ERR_RSP_INVALID_CHR_IN_HEADER ""
+#define ERR_RSP_HEADER_WITHOUT_COLON ""
 #endif
 
 /**
-- 
cgit v1.2.3

